packages/ui/components/AppCard.tsx

```diff
@@ -0,0 +1,41 @@
+import { Info, ServiceInfoBadge } from './ServiceInfoBadge';
+
+interface AppCardProps {
+  isActive?: boolean;
+  name: string;
+  description: string;
+  badges: Info[];
+  isClient: boolean;
+  isServer: boolean;
+  className?: string;
+}
+
+export const AppCard = ({isActive = false, name, description, badges, className, isClient, isServer}:AppCardProps) => {
+  return (
+    <>
+      <div
+        className={`bg-gray-800 border-gray-800 rounded-lg w-[523px] border-2 ${className} ${
```

Comment:
```suggestion
        className={`bg-gray-800 border-gray-800 rounded-lg max-w-[523px] w-full border-2 ${className} ${
```
We can make it responsive for smaller screens.